p4src/v1model/include/control/int.p4

```diff
@@ -297,7 +297,7 @@ control IntEgress (inout v1model_header_t          hdr_v1model,
         fabric_md.int_report_md.mirror_type = FabricMirrorType_t.INT_REPORT;
         fabric_md.int_report_md.report_type = fabric_md.bridged.int_bmd.report_type;
         fabric_md.int_report_md.ig_port = fabric_md.bridged.base.ig_port;
-        fabric_md.int_report_md.eg_port = (PortId_t)fabric_v1model.recirc_preserved_egress_port;
+        fabric_md.int_report_md.eg_port = (PortId_t)standard_md.egress_port;
```

Comment:
based on the logic from fabric-tna, we use the egress port number from intrinsic metadata, which is the standard metadata of v1model

Reply:
@EmanueleGallone 👆does this change make sense?

Reply:
Yes it does, because `init_int_metadata` is performed BEFORE cloning/recirculating the packet. the `standard_md.egress_port` information is lost AFTER cloning/recirculating the packet.
This is also confirmed by the CI, otherwise I would've expected some INT tests to fail when asserting the original packet's egress port.